src/js/components/SchemaForm.js

```diff
@@ -53,6 +53,13 @@ class SchemaForm extends React.Component {
     this.setState({renderGemini: true});
   }
 
+  componentWillUnmount() {
+    // Unschedule all validation if component unmounts.
+    if (this.timer) {
+      clearTimeout(this.timer);
```

Comment:
The util debounce only debounces a function timeout but it is not possible to just cancel a timeout if the function ain't needed anymore e.g. on `unmount`.